src/main/java/com/softserve/teamproject/entity/Student.java

```diff
@@ -36,6 +36,22 @@
   @Column(name = "image")
   private byte[] image;
 
+  @Column(name = "image_name")
+  @Pattern(regexp = ".+(\\.jpg|\\.jpeg|\\.tiff|\\.png)",
+      message = "Sorry, you should use only allowed file types:"
```

Comment:
i ll solve this problem in my pr with tests.
Do this as I did in CopyPasteScheduleWrapper.